packages/postcss-discard-unused/src/index.js

```diff
@@ -94,26 +96,27 @@ function pluginCreator(opts) {
         OnceExit(css, { list }) {
           const { comma, space } = list;
           css.walk((node) => {
-            const { type, prop, selector, name } = node;
+            const { type } = node;
```

Comment:
Need to destructure these later as right now  TypeScript cannot know the node type.